x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/components/empty_state.test.tsx

```diff
@@ -25,7 +27,7 @@ describe('EmptyState', () => {
 
     expect(wrapper.find('h2').text()).toEqual('Create a schema');
     expect(wrapper.find(EuiButton).prop('href')).toEqual(
-      expect.stringContaining('#indexing-documents-guide-schema')
+      expect.stringContaining(`${docLinks.appSearchIndexingDocsSchema}`)
```

Comment:
```suggestion
      expect.stringContaining(docLinks.appSearchIndexingDocsSchema)
```
No need for a template literal here